scratch3/style.js

```diff
@@ -567,6 +400,385 @@ export default class Style {
+            d: "M36.498 8.748c0 .464-.141.874-.433 1.165l-19.742 19.68c-2.131 2.111-4.673 3.793-7.572 5.01L4.5 36.381l.974-2.317 1.925-.808c2.899-1.218 5.441-2.899 7.572-5.01l19.742-19.68c.292-.292.432-.702.432-1.165 0-.647-.27-1.4-.779-2.123.249.172.498.377.736.614.898.905 1.396 1.983 1.396 2.856",
+            fill: "#0b8e69",
+            opacity: ".15",
+          }),
+          SVG.el("path", {
+            d: "M18.45 12.831a.904.904 0 1 1-1.807 0 .904.904 0 0 1 1.807 0z",
+            fill: "#0b8e69",
+          }),
+        ]),
+        {
+          id: "sb3-penBlock-high-contrast",
+          stroke: "#0b8e69",
+          fill: "none",
+          "stroke-linejoin": "round",
+        },
+      ),
+      SVG.setProps(
+        SVG.group([
+          SVG.el("circle", {
+            opacity: 0.25,
+            cx: 32,
+            cy: 16,
+            r: 4.5,
+          }),
+          SVG.el("circle", {
+            opacity: 0.5,
+            cx: 32,
+            cy: 12,
+            r: 4.5,
+          }),
+          SVG.el("circle", {
+            opacity: 0.75,
+            cx: 32,
+            cy: 8,
+            r: 4.5,
+          }),
+          SVG.el("circle", {
+            cx: 32,
+            cy: 4,
+            r: 4.5,
+          }),
+          SVG.el("path", {
+            d: "M22.672 4.42l-6.172 4V6.1c0-2.01-1.563-3.6-3.5-3.6H4.1C2.076 2.5.5 4.076.5 6.1V14c0 1.927 1.584 3.512 3.6 3.6H13c1.902 0 3.5-1.653 3.5-3.6v-2.283l6.257 3.754.097.075c.02.02.098.054.146.054.267 0 .5-.217.5-.5V4.8c0 .037-.056-.094-.129-.243-.145-.242-.43-.299-.7-.137z",
+            fill: "#000",
+            "stroke-linejoin": "round",
+          }),
+        ]),
+        {
+          id: "sb3-videoBlock-high-contrast",
+          stroke: "#0b8e69",
+          fill: "#FFF",
+          "stroke-opacity": 0.15,
+        },
+      ),
+      SVG.setProps(
+        SVG.group([
+          SVG.el("path", {
+            d: "M25.644 20.5c-1.667 1.937-4.539 3.429-5.977 3.429a1.25 1.25 0 0 1-.557-.137c-.372-.186-.61-.542-.61-1.03 0-.105.017-.207.05-.308.076-.236.624-.986.727-1.173.27-.484.462-1.075.566-1.865A8.5 8.5 0 0 1 24 3.5h4a8.5 8.5 0 1 1 0 17h-2.356z",
+            fill: "#FFF",
+            stroke: "#0b8e69",
+          }),
+          SVG.el("path", {
+            d: "M15.5 21.67c0-1.016-1.494-1.586-2.387-.782l-2.7 2.163A5.958 5.958 0 0 1 6.7 24.33h-.4c-1.035 0-1.8.69-1.8 1.573v4.235c0 .883.765 1.572 1.8 1.572h.4c1.458 0 2.754.423 3.82 1.287l2.598 2.161c.908.75 2.382.188 2.382-.876V21.67z",
+            fill: "#000",
+          }),
+        ]),
+        {
+          id: "sb3-ttsBlock-high-contrast",
+          "stroke-opacity": 0.15,
+        },
+      ),
+
+      // The original icon is in PNG, but the high contrast version uses SVG.
+      // For consistency we use PNG in both places.
+      // https://github.com/scratchfoundation/scratch-gui/blob/beta/src/lib/themes/high-contrast/extensions/translateIcon.svg
+      // Exported via Inkscape and compressed
```

Comment:
I think I would slightly prefer to use SVG wherever possible, I don't feel strongly about it though.